discovery-provider/src/queries/get_challenges.py

```diff
@@ -0,0 +1,127 @@
+from functools import reduce
+from collections import defaultdict
+# pylint: disable=E0611
```

Comment:
why did we have to add this no-name thing? should it be a global lint config?

Reply:
This is a holdover until I can get us on the latest linter, which doesn't have this error. I have it working in another branch, but one thing at a time heh

Reply:
i'll add a comment tho